### PR TITLE
[DOC] Add link to "double backward" from "extending pytorch" page

### PR DESCRIPTION
**Pull request by felixdivo**
It is probably the most user friendly to link to that (lesser known?) feature.

### CODE REVIEW

File: docs/source/notes/autograd.rst [hunk 1]
```diff
@@ -353,8 +353,8 @@ potential parallelism, it's important that we ensure thread safety on CPU with
 parallel backwards that share part/whole of the GraphTask.
 
 Custom Python ``autograd.function`` is automatically thread safe because of GIL.
-for built-in C++ Autograd Nodes(e.g. AccumulateGrad, CopySlices) and custom
-``autograd::Function``, the Autograd Engine uses thread mutex locking to protect
+For built-in C++ Autograd Nodes (e.g. AccumulateGrad, CopySlices) and custom
```
Comment (albanD) [inline docs/source/notes/autograd.rst:356]: nit: could you update the line above to make it `autograd.Function` as well?

**Review by albanD — COMMENTED** [review]
Thanks for fixing this!

File: docs/source/notes/autograd.rst (hunk 1 above)
Comment (felixdivo) [reply docs/source/notes/autograd.rst:356]: Sure :)

**Review by soulitzer — APPROVED** [review]

**Comment by felixdivo** [comment]
> You've committed this PR, but

Uhm, ... did it work? I don't quite understand the merge process in this repo.

**Pull request closed without merging**